libmisc/copydir.c

```diff
@@ -771,12 +772,16 @@ static int copy_file (const char *src, const char *dst,
 	        && (errno != 0))
 #endif				/* WITH_ATTR */
 	   ) {
+		if (ofd > 0) {
```

Comment:
unlikely case, but this should be >= 0 ?

Reply:
Fixed